### DISCUSSION TITLE
vhdx from sd card

### DISCUSSION THREAD

**Discussion by JovanovicA** [RedGreenBlue09/WFAv7_Installer#8] [Ideas]
is it posible to boot vhdx from sd card without data partition, Thanks in advance

**Comment by RedGreenBlue09** [comment]
As I know, the Lumia firmware itself don't allow loading an OS from sdcard. I don't think you can exploit the firmware by using VHDX though, but you can try it yourself, manually.

**Reply by JovanovicA** [reply]
Ok thanks for your quick reponse, and best regards
I try something similar with cmdinjector and bcdedit, but for now without succseed
another time best regards
is any diferente if i use VHD orVHDX?

**Reply by RedGreenBlue09** [reply]
About performance, it's basically the same, but VHDX is superior in various areas. Also keep in mind VHD is from 2003.
